bin/prepare-release.sh

```diff
@@ -90,6 +90,14 @@ run $SED -i.bak -E \
     "${WORKSPACE_ROOT}/${FILE}"
 run rm "${WORKSPACE_ROOT}/${FILE}.bak"
 
+### GLEAN-PREVIEW ###
+
+FILE=glean-core/preview/Cargo.toml
+run $SED -i.bak -E \
+    -e "/glean-core/!b;n;n;s/version = \"[0-9a-z.-]+\"/version = \"${NEW_VERSION}\"/" \
```

Comment:
glean-preview is currently at v0.0.4.
That makes it explicitly unstable (`< 1.0.0`, as per semver). I'd like to keep it that way.

Reply:
Oh wait, this only changes the version it depends on, right? Then of course, go ahead!

Reply:
Yes -- just to update the version of glean-core that glean-preview depends on.

Reply:
Can we add a comment on what the sed is supposed to do?

Reply:
Sure thing.